src/components/authors/authorPage.js

```diff
@@ -31,7 +32,7 @@ var AuthorPage = React.createClass({
 		return (
 			<div>
 				<h1>Authors</h1>
-				<Link to="addAuthor" className="btn btn-default">Add Author</Link>
+				<Link to="author" className="btn btn-default">Add Author</Link>

```

Comment:
Again, note that the `to` prop in React Router 4 accepts a path, **not** a route name. So changed to specify the actual desired route.